src/lib/ares__buf.c

```diff
@@ -229,9 +229,9 @@ ares_status_t ares__buf_append(ares__buf_t *buf, const unsigned char *data,
   return ARES_SUCCESS;
 }
 
-ares_status_t ares__buf_append_byte(ares__buf_t *buf, unsigned char byte)
+ares_status_t ares__buf_append_byte(ares__buf_t *buf, unsigned char b)
```

Comment:
This could be fixed by adding `-DWATT32_NO_OLDIES` somewhere (in `Makefile.msvc`?).

Reply:
Eh, it was a simple enough change that it doesn't hurt anything.  Easier than needing to document that someone compiling for WATT32 needs to also define that.